package.json

```diff
@@ -2,23 +2,35 @@
   "name": "redux-logger",
   "version": "3.0.1",
   "description": "Logger for Redux",
-  "main": "lib/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "src/index.js",
+  "jsnext:main": "src/index.js",
   "scripts": {
-    "lint": "$(npm bin)/eslint src",
-    "test": "NODE_ENV=development npm run lint && npm run spec",
-    "test:production": "NODE_ENV=production npm run lint && npm run spec",
-    "spec": "NODE_PATH=src nyc --all --silent --require babel-core/register mocha --plugins transform-inline-environment-variables --recursive spec/*.spec.js",
-    "spec:watch": "NODE_ENV=development npm run spec -- --watch",
+    "lint": "eslint src",
+    "test": "npm run lint && npm run spec",
+    "spec": "nyc --all --silent --require babel-core/register mocha --plugins transform-inline-environment-variables --recursive spec/*.spec.js",
+    "spec:watch": "npm run spec -- --watch",
     "coverage": "nyc report",
-    "coverage:html": "nyc report --reporter=html && (http-server -p 8077 ./coverage & open-url http://localhost:8077/)",
+    "coverage:html": "nyc report --reporter=html && http-server -p 8077 ./coverage -o",
     "coverage:production": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "clean": "$(npm bin)/rimraf dist lib",
-    "build:lib": "$(npm bin)/babel src --out-dir lib",
-    "build:umd": "LIBRARY_NAME=reduxLogger NODE_ENV=development $(npm bin)/webpack src/index.js dist/index.js --config webpack.build.js",
-    "build:umd:min": "LIBRARY_NAME=reduxLogger NODE_ENV=production $(npm bin)/webpack -p src/index.js dist/index.min.js --config webpack.build.js",
-    "build": "npm run build:lib && npm run build:umd && npm run build:umd:min",
-    "precommit": "npm run test",
-    "prepublish": "npm run clean && npm run test:production && npm run build"
+    "clean": "rimraf dist",
+    "build": "npm run rollup && npm run uglify",
+    "precommit": "npm test",
+    "prepublish": "npm run clean && npm test && npm run build",
+    "uglify": "uglifyjs dist/index.umd.js -cm -o dist/index.umd.min.js",
+    "rollup": "node rollup.config.js"
+  },
```

Comment:
Can move scripts as they appears? For example, `build` run `rollup`, so `rollup` must be higher than `build`